nnoir-onnx/test/test_reduce_mean.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+
+import onnx
+from onnx.helper import make_node, make_graph, make_model, make_tensor_value_info, make_opsetid
+from onnx import TensorProto
+from onnx.numpy_helper import from_array
+
+from util import Base
+
+info = make_tensor_value_info
+
+
+def test_reduce_mean_00():
+    '''
+    opset version >= 1
+    '''
+    class ReduceMeanTester(Base):
+        def create_onnx(self) -> onnx.ModelProto:
+            node = make_node("ReduceMean", inputs=["v0"], outputs=["v1"])
+            inputs = [info("v0", TensorProto.FLOAT, (1, 3, 4, 5))]
+            outputs = [info("v1", TensorProto.FLOAT, (1, 1, 1, 1))]
+
+            graph = make_graph([node], "add_graph", inputs, outputs)
+            model = make_model(graph)
+            return model
+
+    v0 = np.random.rand(1, 3, 4, 5).astype(np.float32)
+
+    outputs = ["v1"]
+    ReduceMeanTester({"v0": v0}, outputs).run()
+
+
+def test_reduce_mean_01():
+    '''
+    opset version >= 1
+    '''
+    class ReduceMeanTester(Base):
+        def create_onnx(self) -> onnx.ModelProto:
+            node = make_node("ReduceMean", inputs=["v0"], outputs=["v1"], keepdims=0)
+            inputs = [info("v0", TensorProto.FLOAT, (1, 3, 4, 5))]
+            outputs = [info("v1", TensorProto.FLOAT, [])]  # the shape is scalar
+
+            graph = make_graph([node], "add_graph", inputs, outputs)
+            model = make_model(graph)
+            return model
+
+    v0 = np.random.rand(1, 3, 4, 5).astype(np.float32)
+
+    outputs = ["v1"]
+    ReduceMeanTester({"v0": v0}, outputs).run()
+
+
+def test_reduce_mean_02():
+    '''
+    opset version >= 1
+
+    Currently, failed to packing as msgpack
```

Comment:
What is the meaning?
Currently all tests are passed including this.

Reply:
It means can we del one in`test_reduce_sum.py` ?

Reply:
If your test use `Base` in `test/util.py`, dumping `.nnoir` to a file is already tested.
https://github.com/Idein/nnoir/blob/support-reducemean/nnoir-onnx/test/util.py#L36

Yes, I think we can delete the line.